Classes/rental.rb

```diff
@@ -0,0 +1,8 @@
+class Rental
+  attr_accesor :date, :book, :person
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+end
```

Comment:
- [ ] # *The rental should be added itself to the @book achieving the 'belongs-to' association
``book.rentals << self``
- [ ] # *The rental is adding itself to the @person the 'belongs-to' association
`person.rentals << self`

Reply:
Thanks Man